motion-1/Overview.bs

```diff
@@ -92,68 +92,140 @@ In addition to the property-specific values listed in their definitions, all pro
+Specifies the <a>path</a> the element gets positioned at. The element's exact position on the path is determined by the 'offset-distance' property. A <dfn>path</dfn> is either a specified path with one or multiple sub-paths or the geometry of a not styled basic shape. Each shape or path must define an <dfn>initial position</dfn> for the computed value of "0" for 'offset-distance' and an <dfn>initial direction</dfn> which specifies the rotation of the object on the initial position.
 
-In this specification a direction (or rotation) of 0 degree is equivalent to the direction of the positive x-axis in the object's <a>local coordinate system</a>. E.g a rotation of 0 degree points to the right side of the UA if the object and its ancestors have no transformation applied.
+In this specification, a direction (or rotation) of 0 degree is equivalent to the direction of the positive y-axis in the object's <a>local coordinate system</a>. E.g a rotation of 0 degree points to the upper side of the UA if the object and its ancestors have no transformation applied.
 
 Values have the following meanings:
 
-<dl dfn-for="motion-path">
-  <dt><<basic-shape>></dt>
+<dl dfn-for="offset-path">
+  <dt><<angle>></dt>
   <dd>
-    The motion path is a basic shape as specified in CSS Shapes [[!CSS-SHAPES]]. The initial position and the initial direction for basic shapes are defined as follows:
+    The path is a closed line segment that starts from the current position of the element and has the degree of the angle as specified in <<angle>> from the Y-axis. The path ends at the edge of the containing block.
     <dl>
-      <dt><<circle()>></dt>
-      <dt><<ellipse()>></dt>
-      <dd>The <a>initial position</a> is defined by the point where a virtual tangent to the circle/ellipse would reach the top vertical position. The <a>initial direction</a> is 90 degrees.</dd>
-      <dt><<inset()>></dt>
-      <dd>The <a>initial position</a> is defined by the left top corner of the rectangle. The <a>initial direction</a> is 0 degree.</dd>
-      <dt><<polygon()>></dt>
-      <dd>The <a>initial position</a> is defined by the first coordinate pair of the polygon. The <a>initial direction</a> is defined by the vector connecting the initial position with the next following coordinate pair that isn't equal to the initial position. If there is no such unequal coordinate pair, the initial direction is defined with 0 degree.</dd>
+        <dt><dfn>contain</dfn></dt>
+        <dd>
+           Adjust the value of 'offset-distance' when the element goes out of the path. When overflowing occurs, the distance value is reduced, until the end point of the path is on the edge of the element.
```

Comment:
Is this a clipping operation? If so, the use the word clip:
"When the value of 'offset-distance' would select a point on the offset path which is outside the edge of the containing element, the value is instead clipped so that the selected point lies on the edge of the element'